apitable.go

```diff
@@ -23,7 +23,12 @@ package docx
 // AddTable add a new table to body by col*row
 //
 // unit: twips (1/20 point)
-func (f *Docx) AddTable(row int, col int) *Table {
+func (f *Docx) AddTable(
+	row int, 
+	col int, 
+	tableWidth int64, 
+	borderColors [6]string, 
```

Comment:
You can use a struct with 6 direction fields to make the parameter more human-friendly.